conans/client/detect.py

```diff
@@ -189,7 +189,7 @@ def detect_defaults_settings(output):
     """ try to deduce current machine values without any
     constraints at all
     """
-    output.writeln("\nIt seems to be the first time you run conan", Color.BRIGHT_YELLOW)
+    output.writeln("\nIt seems to be the first time you've ran conan", Color.BRIGHT_YELLOW)
```

Comment:
Excuse my ignorance (not native speaker, willing to learn), if this command is printed the first conan command, and is practically one of the first lines that is printed, wouldn't make more sense something like "are running"? You "have ran" conan, sounds like a past action, an action that has already concluded/finished, and this doesn't seem the case, conan is actively running, and still many things to execute.